pkg/flatbuffers/Makefile.include

```diff
@@ -0,0 +1,12 @@
+INCLUDES += -I$(PKGDIRBASE)/flatbuffers/include
+
+FLATC ?= flatc
+
+ifneq (0,$(shell which flatc 2>&1 > /dev/null ; echo $$?))
+  FLATC = $(RIOTTOOLS)/flatc/flatc
+  $(call target-export-variables,all,FLATC)
+endif
+
+ifneq (,$(filter arch_esp32 arch_esp8266,$(FEATURES_USED)))
+  CXXEXFLAGS += -std=c++11
```

Comment:
Why is this needed?

Reply:
It seems that cpp11 support must be enabled explicitly on theses platforms, I pushed a chage that enables it globally for the whole package (with a dependency to cpp11-compat).